src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountDocumentServiceImplTest.java

```diff
@@ -0,0 +1,66 @@
+package edu.cornell.kfs.pdp.batch.service.impl;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import edu.cornell.kfs.pdp.batch.fixture.StandardEntryClassConversionFixture;
+import edu.cornell.kfs.pdp.batch.fixture.StandardEntryClassConversionFixture.ExpectFailureData;
+import edu.cornell.kfs.pdp.batch.fixture.StandardEntryClassConversionFixture.ExpectPDPSuccessData;
+
+public class PayeeACHAccountDocumentServiceImplTest {
+    
+    private PayeeACHAccountDocumentServiceImpl payeeACHAccountDocumentService;
+    
+    @BeforeEach
+    public void setUp() throws Exception {
+        payeeACHAccountDocumentService = new PayeeACHAccountDocumentServiceImpl();
+    }
+    
+    @AfterEach
+    public void tearDown() throws Exception {
+        payeeACHAccountDocumentService = null;
+    }
+    
+    static Stream<Arguments> goodACHAccountTypesForValidation() {
+        return Stream.of(
+                StandardEntryClassConversionFixture.ExpectPDPSuccessData.WORKDAY_PERSONAL_CHECKING_TEST,
+                StandardEntryClassConversionFixture.ExpectPDPSuccessData.WORKDAY_PERSONAL_SAVINGS_TEST)
+                .map(Arguments::of);
+    }
+    
+    @ParameterizedTest
+    @MethodSource("goodACHAccountTypesForValidation")
+    void testGoodAccountTypeToStandardEntryClassConversion(ExpectPDPSuccessData testFixture) {
+        assertEquals(payeeACHAccountDocumentService.determineStandardEntryClass(testFixture.achBankAccountTypeBeingTested),
+                testFixture.standardEntryClassExpected,
```

Comment:
In `assertEquals()` assertions, the expected value should be the _FIRST_ argument and the actual value should be the _SECOND_ argument. The same goes for the related test in the PaymentWorks test class.